swift/aperture/DeviceList.swift

```diff
@@ -10,8 +10,8 @@ private func enableDalDevices() {
   CMIOObjectSetPropertyData(CMIOObjectID(kCMIOObjectSystemObject), &property, 0, nil, UInt32(sizeOfAllow), &allow)
 }
 
-final class DeviceList {
-  func audio() -> [Dictionary<String, String>] {
+struct DeviceList {
+  static func audio() -> [Dictionary<String, String>] {
```

Comment:
Nice shortcut!

Btw, unrelated, but what do you think about exposing the `DeviceList` as a static property on  `Recorder`? So `Recorder.devices.audio()`. I kinda feel like they're very much related and eventually I want to expose all this as a `Aperture` framework.